sections/editing.include

```diff
@@ -2098,19 +2098,21 @@ Editing:
     The element with ID "a" in the following example would be the one used to determine if the
     word "Hello" is checked for spelling errors. In this example, it would not be.
 
-    <pre highlight="html">
-  &lt;div contenteditable="true">
-  &lt;span spellcheck="false">Hell&lt;/span>&lt;em>o!&lt;/em>
-  &lt;/div>
-  </pre>
+    <xmp highlight="html">
+      <div contenteditable="true">
+        <span id="a" spellcheck="false">Hello</span>
+      </div>
+    </xmp>
 
     The element with ID "b" in the following example would have checking enabled (the leading
     space character in the attribute's value on the <{input}> element causes the attribute
     to be ignored, so the ancestor's value is used instead, regardless of the default).
 
-    <pre class="bad">&lt;p spellcheck="true">
-  &lt;label>Name: &lt;input spellcheck=" false">&lt;/label>
-  &lt;/p></pre>
+    <xmp class="bad">
+      <p spellcheck="true">
+        <label>Name: <input id="b" spellcheck=" false"></label>
```

Comment:
should the space before false be removed?

Reply:
Ah, that's one of the deliberately broken ones.

> The element with ID "b" in the following example would have checking enabled (the leading space character in the attribute's value on the <{input}> element causes the attribute to be ignored, so the ancestor's value is used instead, regardless of the default).